### PR TITLE
WDP200303-34

### PR DESCRIPTION
**Pull request by jurecki**
Feedback section - functionality
In this task, I have to add a slider animation in the "Feedback" using SwipeComponent

### CODE REVIEW

**Comment by Varenthein** [comment]
@ichtys77 Review request :)

**Comment by jurecki** [comment]
I used this solution: https://stackoverflow.com/questions/16805684/javascript-disable-text-select

File: src/components/features/NewFurniture/NewFurniture.js [hunk 1]
```diff
@@ -153,15 +153,21 @@ class NewFurniture extends React.Component {
...
+            <div className='row'>
+              {feedback
+                .slice(activePageFeedback * 1, (activePageFeedback + 1) * 1)
+                .map(item => (
+                  <div key={item.id} className='col-12'>
+                    <ClientFeedback {...item} />
+                  </div>
+                ))}
+            </div>
+          </SwipeComponent>
```
Comment (ichtys77) [inline src/components/features/NewFurniture/NewFurniture.js]: for now the feedback component is a part of NewFurniture so it's not convinient for change its position on Homepage. Maybe You can manage a new, seperate component?

Comment (jurecki) [reply src/components/features/NewFurniture/NewFurniture.js]: Very good idea

**Pull request merged**